h/tasks/indexer.py

```diff
@@ -49,6 +50,34 @@ def reindex_user_annotations(userid):
         log.warning("Failed to re-index annotations into ES6 %s", errored)
 
 
+@celery.task
+def reindex_annotations_in_date_range(start_date, end_date, max_annotations=250000):
+    """Re-index annotations from Postgres to Elasticsearch in a date range.
+
+    :param start_date: Begin at this time (greater or equal)
+    :param end_date: End at this time (less than or equal)
+    :param max_annotations: Maximum number of items to process overall
+
+    """
+    log.info(f"Re-indexing from {start_date} to {end_date}...")
```

Comment:
Oops my fault, this should not be interpolating.

```suggestion
    log.info(f"Re-indexing from %s to %s...", start_date, end_date)
```